src/CommitmentsV2/SFA.DAS.CommitmentsV2/Services/CohortDomainService.cs

```diff
@@ -141,7 +141,21 @@ public async Task<Cohort> UpdateDraftApprenticeship(long cohortId, DraftApprenti
             return cohort;
         }
 
-        private ICohortOriginator GetCohortOriginator(Party originatingParty, Provider provider,  AccountLegalEntity accountLegalEntity)
+        public async Task<Cohort> DeleteDraftApprenticeship(long cohortId, long apprenticeshipId, UserInfo userInfo, CancellationToken cancellationToken)
+        {
+            var db = _dbContext.Value;
+
+            var cohort = await db.Cohorts
+                                .Include(c => c.Apprenticeships)
+                                .SingleAsync(c => c.Id == cohortId, cancellationToken: cancellationToken);
+
+            AssertHasApprenticeshipId(cohortId, apprenticeshipId);
```

Comment:
We seem to have AssertHasApprenticeshipId() declared twice. It could be combined into a single function. I'm also a little confused about what it's purpose is.

Is it checking the apprenticeship exists in the cohort prior to deleting it? That was my assumption. But it simply checks is the Id is less than 1.

Reply:
Agreed, the private static function doesn't do much. I will remove it and merge the code in the other function.